library/src/main/scala/scala/scalajs/js/typedarray/Float32Array.scala

```diff
@@ -18,7 +18,7 @@ class Float32Array private extends TypedArray[Float, Float32Array] {
   def this(typedArray: TypedArray[_, _]) = this()
 
   /** Creates a new Float32Array with the elements in the given array */
-  def this(array: js.Array[_]) = this()
+  def this(array: js.Iterable[_]) = this()
```

Comment:
With this new constructor, is the constructor taking a `TypedArray` still necessary?

Shouldn't we simply add an overload with `js.Iterable` rather than modifying this one? How is the spec formulated?